src/content/docs/contributing/modules.md

```diff
@@ -360,6 +360,12 @@ curl -fsSL https://code.askimed.com/install/nf-test | bash
 The install step will copy the nf-test executable file in the current directory. If you want to use it globally, move the nf-test file to a directory accessible by your $PATH variable.
 :::
 
+- Or install via conda and make sure to update to the latest version
+
+```bash
+conda install -c bioconda nf-test
```

Comment:
oh, didn't know about that. this should be the default imo, because the other install script just downloads it to the current folder and doesn't add it to the path

Reply:
this doesn't get the latest version and im afraid ppl might forget to update....is that ok ?

Reply:
I try to update the recipe whenever a new release happens. Also I think not everyone install nf-test everytime they create a test, right?

Reply:
having it on conda also makes CI easier. So with this we have more pressure to keep it updated there 🙂

Reply:
thats right